controllers/enable_mesh.go

```diff
@@ -136,3 +135,18 @@ func (r *OpenshiftServiceMeshReconciler) findIstioIngress(ctx context.Context) (
 
 	return routes, nil
 }
+
+func (r *OpenshiftServiceMeshReconciler) findAppDomain(ctx context.Context) (string, error) {
```

Comment:
Is fetching `Ingress` resource the only wahy to find it?

Reply:
Maybe not the only way to find it, but it's the controlling value location.